File: backoffice/src/shared/domain/exceptions/ServerInternalErrorException.py
"""
 *
 * Libraries 
 *
"""

from .DomainException import DomainException

"""
 *
 * Class 
 *
"""

class ServerInternalErrorException( DomainException ):

    """
     *
     * Constants 
     *
    """

    __SERVER_INTERNAL_ERROR = 103

    """
     *
     * Methods 
     *
    """

    def __init__( self ) -> object:
        super().__init__( 
            self.__SERVER_INTERNAL_ERROR,
            'Internal server error occurred while the request was being processed'
        )

File: preparation/src/shared/domain/exceptions/ServerInternalErrorException.cs
/* 
 * 
 * Class
 *
*/

namespace preparation.src.shared.domain {

    public class ServerInternalErrorException : DomainException {

        /* 
         * 
         * Constants
         *
        */

        private const int SERVER_INTERNAL_ERROR = 103;

        /* 
         * 
         * Methods
         *
        */

        public ServerInternalErrorException() : base( 
            SERVER_INTERNAL_ERROR,
            "Internal server error occurred while the request was being processed" 
        ) {
        }

    }

}